src/elements/title/title.component.ts

```diff
@@ -16,6 +16,10 @@ export
 @customElement('sbb-title')
 class SbbTitleElement extends SbbNegativeMixin(SbbTitleBase) {
   public static override styles: CSSResultGroup = [SbbTitleBase.styles, style];
+
+  /** Visual level for the title. Optional, if not set, the value of level will be used. */
+  @property({ attribute: 'visual-level', reflect: true })
```

Comment:
the @property maybe doesn't need to be copied?

Reply:
Apparently, it's needed: i see tests failing otherwise